### PR TITLE
docs: Add tutorial handout to Quick Ref page

### PR DESCRIPTION
**Pull request by sarina**

### CODE REVIEW

File: source/documentors/references/quick_reference_rst.rst [hunk 1]
```diff
@@ -4,8 +4,13 @@ Quick Reference: Writing RST
...
 can use it to write Open edX documentation.
 
-For more details check out the `RST Primer`_
+.. seealso::
+
+   `A one-page reference document`_ (PDF)
+
+   For a deeper introduction, check out the `RST Primer`_
 
+.. _A one-page reference document: https://docs.openedx.org/en/latest/documentors/references/RST_quick_reference.pdf
```
Comment (sarina) [inline source/documentors/references/quick_reference_rst.rst:13]: @feanil I don't know how to link this PDF. I added to the directory, but https://docsopenedxorg--286.org.readthedocs.build/en/286/documentors/references/RST_quick_reference.pdf doesn't show up. Is there another place to put a PDF / another way to display it?

File: source/documentors/references/quick_reference_rst.rst [hunk 2]
```diff
@@ -4,8 +4,13 @@ Quick Reference: Writing RST
 RST is very powerful and flexible.  Below we provide a quick guide for how you
 can use it to write Open edX documentation.
 
-For more details check out the `RST Primer`_
+.. seealso::
```
Comment (feanil) [inline source/documentors/references/quick_reference_rst.rst:7]: Seealso directives are usually at the bottom, and there is already one there that this link can be added to.  It feels weird to send them away from this page as soon as they have arrived to it.  I think you should just add this to the see-also section at the bottom.

File: source/documentors/references/quick_reference_rst.rst (hunk 1 above)
Comment (feanil) [reply source/documentors/references/quick_reference_rst.rst:13]: I think the [:download:](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#referencing-downloadable-files) role is probably the way to go.

**Review by feanil — CHANGES_REQUESTED** [review]

File: source/documentors/references/quick_reference_rst.rst (hunk 1 above)
Comment (sarina) [reply source/documentors/references/quick_reference_rst.rst:13]: Would I put the role here, or above? EG,

```
.. _A one-page reference document: :download:https://docs.openedx.org/en/latest/documentors/references/RST_quick_reference.pdf
```

or above, eg

```
:download:`A one-page reference document`_ (PDF)
```

I'll also change it back to a relative link, which wasn't working before, and see if the directive helps

**Comment by sarina** [comment]
The page & download now works! https://docsopenedxorg--286.org.readthedocs.build/en/286/documentors/references/quick_reference_rst.html

**Review by feanil — APPROVED** [review]

**Pull request merged**